hbase-client/src/main/java/org/apache/hadoop/hbase/ipc/NettyRpcClient.java

```diff
@@ -39,14 +49,20 @@
 @InterfaceAudience.LimitedPrivate(HBaseInterfaceAudience.CONFIG)
 public class NettyRpcClient extends AbstractRpcClient<NettyRpcConnection> {
 
+  private static final String HBASE_NETTY_ALLOCATOR_KEY = "hbase.netty.rpcclient.allocator";
+  private static final String POOLED_ALLOCATOR_TYPE = "pooled";
+  private static final String UNPOOLED_ALLOCATOR_TYPE = "unpooled";
+
   final EventLoopGroup group;
 
   final Class<? extends Channel> channelClass;
 
   private final boolean shutdownGroupWhenClose;
+  private final AtomicReference<SslContext> sslContextForClient = new AtomicReference<>();
+  private final ByteBufAllocator byteBufAllocator;
 
   public NettyRpcClient(Configuration configuration, String clusterId, SocketAddress localAddress,
-    MetricsConnection metrics) {
+    MetricsConnection metrics) throws SSLContextException, IOException {
```

Comment:
I do not think we will throw these exceptions after refactoring, so just remove the throws here?

Reply:
Done.